app/assets/stylesheets/v3/layout_components/buttons/buttons.scss

```diff
@@ -111,6 +111,44 @@
     }
   }
 
+  &--ghost {
+    @extend .button;
+    display: flex;
+    text-align: left;
+    align-items: center;
+    background-color: $white;
+    border: 3px solid $blue-main-brand;
+    min-height: 4.5rem;
+    padding: $base-padding/2 $base-padding*3 $base-padding/2 $base-padding;
+    color: $blue-main-brand;
+    vertical-align: middle;
```

Comment:
I'm not sure you need this with align-items: center?

Reply:
You are right! I forgot to remove it when I added flex to the buttons. Will fix!